src/diffusers/models/embeddings.py

```diff
@@ -415,9 +415,10 @@ def __init__(
 
         if set_W_to_weight:
             # to delete later
+            del self.weight
             self.W = nn.Parameter(torch.randn(embedding_size) * scale, requires_grad=False)
-
             self.weight = self.W
+            del self.W
```

Comment:
Necessary to mitigate the "tied weight tensors" error from `safetensors`.

Reply:
@sayakpaul curious about codes here, may i ask for more details if possible? ;)